integration/utils/follower.go

```diff
@@ -136,12 +142,16 @@ Loop:
 				if ch, loaded := f.txToChan.LoadAndDelete(tx.Hex()); loaded {
 					txi := ch.(txInfo)
 
+					duration := time.Since(txi.submisionTime)
 					f.logger.Trace().
-						Dur("duration", time.Since(txi.submisionTime)).
+						Dur("durationInMS", duration).
```

Comment:
Why is this in milliseconds, and the prometheus stats in seconds?

Reply:
Sadly, `Dur` uses the  `zerolog.DurationFieldUnit` unit for measuring latencies.  But the time to execution block time is about 9 seconds (this is the number of roundtrips between the block stages multiplied by the fixed length of the sage).  Therefore I felt that measuring value that will likely be always greater than a second in milliseconds was quite silly.  Can switch it back for consistency though.